sdf.md

```diff
@@ -807,7 +813,15 @@ This example defines an Object "foo" that is defined in the default namespace (f
 `#/sdfObject/foo/sdfProperty/bar`, with data of type boolean.
 <!-- we could define a URN-style namespace that looks exactly that way -->
 
-Some of the definitions are also declarations: the definition of the entry "bar" in the property "foo" means that each instance of a "foo" can have zero or one instance of a "bar".  Entries within `sdfProperty`, `sdfAction`, and `sdfEvent`, within `sdfObject` entries, are declarations.  Similarly, entries within an `sdfThing` describe instances of `sdfObject` (or nested `sdfThing`) that form part of instances of the Thing.
+Some of the definitions are also declarations: the definition of the
```

Comment:
Given declarations are the norm, would it make sense to say “Often the definitions are…”.  and note the exception here explicitly?

Reply:
Great point.  See commit below.

Reply:
Great, thanks!